hybrasyl/Objects/Creature.cs

```diff
@@ -302,8 +302,8 @@ public virtual List<Creature> GetTargets(Castable castable, Creature target = nu
                         Logger.Error($"GetTargets: {castable.Name} - intent was for exact clicked target but no target was passed?");
                     else
                         // Heal spels can be cast on players, other spells can be cast on attackable creatures
-                        if ((!castable.Effects.Damage.IsEmpty && target.Condition.IsAttackable) ||
-                        (castable.Effects.Damage.IsEmpty && target is User))
+                        if ((castable.Effects.Damage != null && target.Condition.IsAttackable) ||
+                        (castable.Effects.Heal != null && target is User))
```

Comment:
Heal should never be null. This looks like it was caused by a change made to your castables class.